INFO     homeassistant.loader:loader.py:638 Loaded bouncie from custom_components.bouncie
WARNING  homeassistant.loader:loader.py:576 We found a custom integration bouncie which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
INFO     homeassistant.setup:setup.py:257 Setting up bouncie
INFO     homeassistant.setup:setup.py:308 Setup of domain bouncie took 0.0 seconds
ERROR    homeassistant.config_entries:config_entries.py:558 Error unloading entry Bouncie for bouncie
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/homeassistant/config_entries.py", line 546, in async_unload
    result = await component.async_unload_entry(hass, self)
  File "/home/runner/work/ha-bouncie/ha-bouncie/custom_components/bouncie/__init__.py", line 41, in async_unload_entry
    hass.data[DOMAIN].pop(entry.entry_id)
KeyError: 'bouncie'